src/openrct2-ui/input/KeyboardShortcuts.cpp

```diff
@@ -169,27 +169,15 @@ void KeyboardShortcuts::GetKeyboardMapScroll(const uint8_t* keysState, int32_t*
         if (!keysState[scancode])
             continue;
 
-        if (shortcutKey & SHIFT)
-        {
-            if (!keysState[SDL_SCANCODE_LSHIFT] && !keysState[SDL_SCANCODE_RSHIFT])
-                continue;
-        }
-        if (shortcutKey & CTRL)
-        {
-            if (!keysState[SDL_SCANCODE_LCTRL] && !keysState[SDL_SCANCODE_RCTRL])
-                continue;
-        }
-        if (shortcutKey & ALT)
-        {
-            if (!keysState[SDL_SCANCODE_LALT] && !keysState[SDL_SCANCODE_RALT])
-                continue;
-        }
+        if ((bool)(shortcutKey & SHIFT) != (keysState[SDL_SCANCODE_LSHIFT] || keysState[SDL_SCANCODE_RSHIFT]))
```

Comment:
If you put a comment above here with the following content:
```
// Check if SHIFT is either set in the shortcut key and currently, or not set in the shortcut key and not currently pressed (in other words: check if they match)
```
and add the following line to the correct location in the changelog:
```
- Fix: [#7878] Scroll shortcut keys ignore SHIFT/CTRL/ALT modifiers
```

I'll merge this pull request.

Reply:
I was just about to do it now then push it. Thank you

Reply:
Yeah, sorry about the confusion here. I was merging stuff anyway and thought I would modify it myself after.